transforms/hwx_views.py

```diff
@@ -1,25 +1,44 @@
 {
-    'data': [
-        ('activity', 'activity', '', 'Activity View',
-            'plottable=t&group=type'),
-        ('org', 'org', 'org', 'Organisation View',
-            'a=org:OrganisationalUnit'),
-        ('orgpeople', 'org', 'org', 'Organisation + People View',
-            'a=org:OrganisationalUnit+foaf:Person&group=org'),
-        ('cap', 'cap', 'cap', 'Capability View', 'a=HE/Capability'),
-    ],
-    'lets': {
-        'iri': 'vm:{row[0]}',
+    'data': [{
+        'view_id': 'activity',
+        'geog_id': 'activity',
+        'tile_id': '',
+        'version': '',
```

Comment:
Activity view has no version? Is this not going to break the S3 bucket URL with a double //?